.pullapprove.yml

```diff
@@ -0,0 +1,29 @@
+version: 3
+
+pullapprove_conditions:
+- "'WIP' not in title"
+- "'ci/circleci: build' in statuses.succeeded"
+
+notifications:
+- when: pull_request.opened
+  if: "author_association in ['FIRST_TIMER', 'FIRST_TIME_CONTRIBUTOR', 'CONTRIBUTOR']"
+  comment: >
```

Comment:
do you sure we need the `>` sign?

Reply:
I can see it in their examples... https://docs.pullapprove.com/